widget/index.html

```diff
@@ -799,6 +799,11 @@
         margin-bottom: 0px !important;
     }
 
+    html[safe-area="true"] #pluginsContainer:matches(.layout1, .layout2, .layout8) {
+        padding-bottom: env(safe-area-inset-bottom) !important;
+        padding-bottom: constant(safe-area-inset-bottom) !important;
+    }
+

```

Comment:
Adds padding-bottom on list view layouts to conform to safe area standards for scrolling content